cmd/proxy/actions/readiness.go

```diff
@@ -0,0 +1,21 @@
+package actions
+
+import (
+	"github.com/gobuffalo/buffalo"
+	"github.com/gomods/athens/pkg/storage"
+	"net"
```

Comment:
add space please and separate imports
first imports from Standard library then all external ones (github...)

Reply:
after removing DNS check "net" import will go away